packages/teams-js/src/public/mail/handoff.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Used to interact with mail capability, including opening and composing mail.
```

Comment:
The `@module` comment and every exported item in this file should include the following comment:

```typescript
 * @beta
```

Reply:
This will print a blurb in the dev docs on this capability that looks like this: 

![image](https://github.com/user-attachments/assets/a6230084-ee1a-46af-9955-cf655f90373a)

You can work with PM to decide when the @beta tag gets removed